packages/boot/test/bootstrapTests/benchmarkerator.md

```diff
@@ -0,0 +1,171 @@
+test implementation.  The exception, as with tests, is imports that bring in any
+kind of pre-lockdown functionality that you will be needing to use; however,
+that should be necessary only in very specialized circumstances.
+
+In a manner similar to the way tests are packaged for Ava, a benchmark file can
+actually contain multiple benchmark tests.  Each should be declared with a
+statement of the form:
+
+`bench.addBenchmark(label: string, benchmark: Benchmark);`
+
+where `label` is a string labelling the benchmark (much as individual tests are
+labelled in Ava), and `benchmark` is an object describing the benchmark itself
+(which will be described in detail below)
+
+After defining one or more benchmarks with `addBenchmark` you must then end by
+invoking:
+
+```await bench.run(name);```
+
+where `name` is a string naming the benchmark run for purposes of error and
+result reporting.
+
+#### The `Benchmark` object
+
+The `benchmark` object may have the following properties, all of which are
+optional except for `executeRound`:
+
+`setup?: (context: BenchmarkContext) => Promise<Record<string, unknown> | undefined>`
+
+  An optional async method to perform any pre-run setup that you need to do. It
+  may optionally return a configuration record containing any benchmark-specific
+  information you desire.  This will be provided as part of the context object
+  in subsequent calls into the benchmark.
+
+`executeRound: (context: BenchmarkContext, round: number) => Promise<void>`
+
+  A required async method which is called to actually execute one round of the
+  benchmark.  The `round` argument is the number of the benchmark round
+  that this call to `executeRound` is being asked to execute.
+
+`finish?: (context: BenchmarkContext) => Promise<void>`
+
+  An optional async method to perform any post-run teardown that you need to do.
+
+`rounds?: number`
+
+  The number of benchmark rounds that will be run if not overridden on the
+  command line.  If omitted, it defaults to `1`.
+
+#### The `BenchmarkContext` object
+
+The first parameter to each of the benchmark methods is a context object that
+provides various information about the execution.  It has the properties:
+
+`options: Record<string, string>`
+
+  Named options from the command line (see Command Line below).
+
+`argv: string[]`
+
+  Remaining unparsed arguments from the command line.
+
+`actors: Record<string, SmartWalletDriver>`
+
+  Wallet drivers for the various personae that exist in the test environment.
+  Currently present are `gov1`, `gov2`, and `gov3` (the governance committee);
+  `atom1` and `atom2` (ersatz ATOM-USD price feed oracles); and `alice`, `bob`,
+  and `carol` (arbitrary users to participate in interactions being exercised).
+
+`label: string`
+
+  The benchmark label, from the `addBenchmark` call that defined this benchmark.
+
+`rounds: number`
+
+  The total number of benchmark rounds that are to be executed in this run.
```

Comment:
My thinking on this is that it's establishing a baseline.  Normally you'd specify the number of rounds on the command line (i.e., dynamic), but if you don't it'll use this.  And if you don't specify a round count in the definition, it will default to 1 which is probably what you want.  But I an imagine some very fuzzy thing that you really need to do 100 rounds to get a decent number, and this would give you a place to say that.